example_test.go

```diff
@@ -914,37 +982,53 @@ func ExampleBeginRequest_TxnIsolation() {
 		Limit(1).
 		Iterator(tarantool.IterEq).
 		Key([]interface{}{uint(2001)})
-	resp, err = conn.Do(selectReq).Get()
+	data, err = conn.Do(selectReq).Get()
 	if err != nil {
 		fmt.Printf("Failed to Select: %s", err.Error())
 		return
 	}
-	fmt.Printf("Select out of stream: response is %#v\n", resp.Data)
+	fmt.Printf("Select out of stream: response is %#v\n", data)
 
 	// Select in stream
-	resp, err = stream.Do(selectReq).Get()
+	data, err = stream.Do(selectReq).Get()
 	if err != nil {
 		fmt.Printf("Failed to Select: %s", err.Error())
 		return
 	}
-	fmt.Printf("Select in stream: response is %#v\n", resp.Data)
+	fmt.Printf("Select in stream: response is %#v\n", data)
 
 	// Rollback transaction
 	req = tarantool.NewRollbackRequest()
-	resp, err = stream.Do(req).Get()
+	data, err = stream.Do(req).Get()
 	if err != nil {
 		fmt.Printf("Failed to Rollback: %s", err.Error())
 		return
 	}
-	fmt.Printf("Rollback transaction: response is %#v\n", resp.Code)
+	fmt.Printf("Rollback transaction: response is %#v\n", data)
 
 	// Select outside of transaction
-	resp, err = conn.Do(selectReq).Get()
+	data, err = conn.Do(selectReq).Get()
 	if err != nil {
 		fmt.Printf("Failed to Select: %s", err.Error())
 		return
 	}
-	fmt.Printf("Select after Rollback: response is %#v\n", resp.Data)
+	fmt.Printf("Select after Rollback: response is %#v\n", data)
+}
+
+func ExampleErrorNo() {
+	conn := exampleConnect(dialer, opts)
+	defer conn.Close()
+
+	req := tarantool.NewPingRequest()
+	resp, err := conn.Do(req).GetResponse()
+	if err != nil {
+		fmt.Printf("error getting the response: %s\n", err)
+		return
+	}
+
+	if resp.Header().Error != tarantool.ErrorNo {
+		fmt.Printf("response error code: %s\n", resp.Header().Error)
+	}
```

Comment:
Let's make the example executable:
```suggestion
	if resp.Header().Error != tarantool.ErrorNo {
		fmt.Printf("response error code: %s\n", resp.Header().Error)
	}
	fmt.Prinln("Success.")
	// Output:
	// Success.
```